chart/kubeapps/values.yaml

```diff
@@ -644,6 +644,9 @@ testImage:
 authProxy:
   # Set to true to enable the OIDC proxy
   enabled: false
+  # Set to true if an external auth proxy is setup to provide cookie authentication
+  # at the oauthLoginURI and oauthLogoutURI values below.
```

Comment:
the comment's are not that clear, people may enable both `authProxy.enabled` and `authProxy.externallyEnabled` (mostly because we don't say anything about not setting `enabled` to true). I think the UX would be better if the params are:

```
enabled: false
deployOauth2Proxy: true
```

so if people want to use their own auth proxy they disable that. WDYT?

Reply:
I think it's only the history that makes this tricky. Let me open a new PR with better comments and discuss this there.